lib/journaled.rb

```diff
@@ -69,3 +69,5 @@ def self.tag!(**tags)
     Current.tags = Current.tags.merge(tags)
   end
 end
+
+require 'journaled/audit_log'
```

Comment:
We need access to some of the module methods above at load time, so I stuck the `require` down at the bottom. (Some gems do this, but I admit it's a little awkward. Might be a sign that other parts of this file need to be stuck into their own mixins).